### DISCUSSION TITLE
Does Parcel v2 support Sass partial imports?

### DISCUSSION THREAD

**Discussion by RuyiLi** [parcel-bundler/parcel#6217] [Q&A]
Link for reference: https://sass-lang.com/documentation/at-rules/import#partials

I'm using Bootstrap in a project and I've been trying to import individual SCSS files for certain components. These tend to rely on a variety of utility Sass files, which are stored in their own folders:
![image](https://user-images.githubusercontent.com/16544254/116794248-974c2480-aa99-11eb-8c2e-400d05230e12.png)

However, despite what the Sass docs say regarding underscore-prefixed imports (e.g. `@import 'mixins'` to import every partial file contained within the mixins folder, or `@import 'variables'` to import `_variables.scss`), this behavior seems to be disallowed when using Parcel v2. Instead, I get an error along the lines of 
```
// my-stylesheet.scss
@import 'bootstrap/scss/mixins';       // a folder containing many partial files (see above image)

// console output
@parcel/transformer-sass: Failed to resolve 'bootstrap/scss/mixins' from 'my-stylesheet.scss'
```
Additionally, when importing lone partial files, the following occurs:
```
@import 'bootstrap/scss/variables';    // a single partial file
                                       // note: importing 'bootstrap/scss/_variables' (with the underscore!) works
// console output
@parcel/transformer-sass: Failed to resolve 'bootstrap/scss/variables' from 'my-stylesheet.scss'
```

Is this a bug, or am I doing something incorrectly?

**Comment by mischnic** [comment]
See https://github.com/parcel-bundler/parcel/issues/3704
